apps/test-suite/TestUtils.js

```diff
@@ -61,20 +56,32 @@ export function getTestModules() {
     require('./tests/Random'),
     require('./tests/Payments'),
     require('./tests/AdMobInterstitial'),
-    require('./tests/AdMobBanner'),
-    require('./tests/AdMobPublisherBanner'),
-    require('./tests/AdMobRewarded'),
-    require('./tests/Video'),
-    require('./tests/Permissions'),
-    require('./tests/MediaLibrary'),
-    require('./tests/Notifications'),
-    require('./tests/FBNativeAd'),
-    require('./tests/FBBannerAd'),
-    require('./tests/TaskManager'),
   ];
+  if (!ExponentTest.isInCI) {
+    // Requires interaction (sign in popup)
+    modules.push(require('./tests/GoogleSignIn'));
+    // Popup to request device's location which uses Google's location service
+    modules.push(require('./tests/Location'));
+    // Fails to redirect because of malformed URL
```

Comment:
i would note that this is specifically because of the release channel. We'll need to modify that backend code in order for it to support test-suite published to a non-default release channel